### DISCUSSION TITLE
Failed to find a suitable terminal rofi when open vim

### DISCUSSION THREAD

**Discussion by thien0709** [davatorium/rofi#1852] [(Possible) Issues] [answered]
I'am using Arch. wm Gnome 
When I open neovim, i can't open it and rofi appears "failed to find a suitable terminal"

**Comment by DaveDavenport** [comment, marked as answer]
Install a suitable terminal?  or configure rofi to use your preferred terminal. 

https://github.com/davatorium/rofi/blob/next/doc/rofi-sensible-terminal.1.markdown
